lib/v0_bindings_extensions.rb

```diff
@@ -145,6 +130,38 @@ def summarize(user_uuid:)
     Api::V0::Bindings::GetHighlightsSummaryParameters
   ].each do |klass|
     klass.class_exec do
+
+      def get_highlights(user_id)
+        filters = to_hash.except(:page, :per_page, :sets)
+        filters[:user] = []
+
+        if (sets.blank? || sets.include?('user:me')) && user_id.present?
```

Comment:
this would be a bit easier to understand if the || part were a method and it read  if sets_me && user_id.present?